misc/server/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "libp2p-server"
+version = "0.12.0"
+authors = ["Max Inden <mail@max-inden.de>"]
+edition = "2021"
+rust-version = "1.65.0"
```

Comment:
Use workspace?

Reply:
Thanks.

@thomaseizinger should we use `rust-version = { workspace = true }` across all crates?

https://github.com/search?q=repo%3Alibp2p%2Frust-libp2p+%22rust-version+%3D+%5C%22%22&type=code

Reply:
It is not a breaking change as long as the current Rust version is > 1.64 (I think) where this field was introduced.

Reply:
:+1: https://github.com/libp2p/rust-libp2p/pull/4344